vernac/indschemes.mli

```diff
@@ -35,12 +41,16 @@ val declare_rewriting_schemes : inductive -> unit
     By default it is [false] and some of the eliminators are defined as simple case analysis.
  *)
 
-val do_mutual_induction_scheme : ?force_mutual:bool ->
-  (lident * bool * inductive * Sorts.family) list -> unit
+val do_mutual_induction_scheme : ?force_mutual:bool -> Environ.env
+  -> (Names.Id.t CAst.t * Indrec.dep_flag * Names.inductive * Sorts.family) list -> unit
```

Comment:
May be worth packing this into a record / type ?

Reply:
Done